audio_process.py

```diff
@@ -105,9 +105,8 @@ def get_syllables(spectral_en, total_en, win_step, threshold_per=40,
                                                        mode="same") +
                                      0.5 * global_mean) / 100.0
     else:
-        threshold = threshold_per * (0.5 * np.mean(threshold_buf) +
-                                     0.5 * np.mean(spectral_en)) / 100.0
-
+        threshold = threshold_per * (0.4 * np.mean(threshold_buf[-50:]) + 
```

Comment:
@vassilinast this "50" is measured in windows right? do you think we could change that to be in mseconds so that it is not dependent on the short-term window size and step? I mean do u think we could make it relative to the window size.

Reply:
Actually I think that each element is associated to the buffer size but I am not sure. Anyway the comment is still valid: maybe this 50 should be expressed in seconds